templates/style.scss

```diff
@@ -84,21 +84,33 @@ div.rustdoc {
         }
     }
 
-    // this is actual fix for docs.rs navigation and rustdoc sidebar
-    position: fixed;
-    top: $top-navbar-height;
-    bottom: 0;
-    left: 0;
-    right: 0;
-    display: block;
-    overflow-y: auto;
+    position: relative;
+}
+
+html {
+    height: 100%;
 }
 
 body {
     padding: 0;
     margin: 0;
-    // Since top navbar is fixed, we need to add padding to the body content.
-    padding-top: $top-navbar-height;
+    height: 100%;
```

Comment:
Normally, you don't need this, can you double-check please?

Reply:
Without it, the viewport turns non-scrollable. /: